eclass/kernel-build.eclass

```diff
@@ -107,6 +107,10 @@ if [[ ${KERNEL_IUSE_MODULES_SIGN} ]]; then
 	"
 fi
 
+if [[ ${KERNEL_IUSE_GENERIC_UKI} ]]; then
+	BDEPEND+="generic-uki? ( ${!INITRD_PACKAGES[@]} )"
```

Comment:
Sorry, let's do:

```suggestion
	BDEPEND+="
		generic-uki? ( ${!INITRD_PACKAGES[@]} )
	"
```

so that we don't accidentally concatenate two atoms together.